test/app/modules/view.xql

```diff
@@ -111,10 +111,17 @@ declare function test:custom-model($node as node(), $model as map(*)) {
     $model?('my-model-item')
 };
 
+declare
+    %templates:wrap
+function test:print-from-class($node as node(), $model as map(*)) {
+    'print-from-class'
+};
 let $config := map {
     $templates:CONFIG_APP_ROOT : $test:app-root,
-    $templates:CONFIG_STOP_ON_ERROR: true()
+    $templates:CONFIG_STOP_ON_ERROR: true(),
+    $templates:CONFIG_USE_CLASS_SYNTAX: xs:boolean(request:get-parameter('classLookup', $templates:SEARCH_IN_CLASS))
```

Comment:
I would prefer to have different configurations to be checked as part of the unit tests. (see
https://github.com/eXist-db/templating/blob/c0f6e4b45abfba8e79cc69ef5157515d0a847efa/test/xqs/test-suite.xqm#L61 and following)